packages/dds/matrix/src/matrix.ts

```diff
@@ -996,64 +1001,22 @@ export class SharedMatrix<T = any>
 	/**
 	 * {@inheritDoc @fluidframework/shared-object-base#SharedObjectCore.applyStashedOp}
 	 */
-	protected applyStashedOp(content: any): unknown {
-		this.inFlightRefSeqs.push(this.runtime.deltaManager.lastSequenceNumber);
-		const parsedContent = parseHandles(content, this.serializer);
-		if (
-			parsedContent.target === SnapshotPath.cols ||
-			parsedContent.target === SnapshotPath.rows
-		) {
-			const op = parsedContent as IMergeTreeOp;
-			const currentVector =
-				parsedContent.target === SnapshotPath.cols ? this.cols : this.rows;
-			const oppositeVector =
-				parsedContent.target === SnapshotPath.cols ? this.rows : this.cols;
-			const metadata = currentVector.applyStashedOp(op);
-			const localSeq = currentVector.getCollabWindow().localSeq;
-			const oppositeWindow = oppositeVector.getCollabWindow();
-
-			assert(
-				localSeq > oppositeWindow.localSeq,
-				0x2d9,
-				/* "The 'localSeq' of the vector applying stashed op must > the 'localSeq' of the other vector." */
-			);
-
-			oppositeWindow.localSeq = localSeq;
+	protected applyStashedOp(content: unknown): void {
+		const parsedContent: MatrixSetOrVectorOp<T> = parseHandles(content, this.serializer);
 
-			return metadata;
+		if (parsedContent.type === MatrixOp.set && parsedContent.target === undefined) {
+			if (parsedContent.fwwMode === true) {
```

Comment:
this is a fix, right? Don't see it implemented in previous diff. Looks correct though based on documentation of the method.